VideoApp/Video-InternalTests/Specs/RemoteConfigStoreSpec.swift

```diff
@@ -79,46 +79,46 @@ class RemoteConfigStoreSpec: QuickSpec {
                     }
                     
                     context("when newValue is group") {
```

Comment:
We probably only need 2 of these contexts now to test that room type is stored properly when set to 2 different values. Shouldn't need to test all cases anymore. And just one test to verify codec is set to auto is probably good.

But up to you. These tests may go away eventually anyway as our testing strategy has evolved.

Reply:
This is better coverage so it's staying.